code/modules/surgery/organs/tongue.dm

```diff
@@ -8,6 +8,11 @@
 	attack_verb_simple = list("lick", "slobber", "slap", "french", "tongue")
 	var/list/languages_possible
 	var/say_mod = null
+	/*
+	* Whether the owner of this tongue can taste anything. Being
+	* set to FALSE will mean no taste feedback will be provided.
+	*/
```

Comment:
This isn't the proper format for autodoc comments

Reply:
Is there a method of doing DMDOC multiline variable comments, or do I just use a series of `///` comments?

Reply:
I'm honestly not sure, I believe a series of `///` will break, one liners are best, but if more then one is needed your best bet is the proc comment format